inst/doctest.m

```diff
@@ -285,12 +310,35 @@
 summary.num_tests = 0;
 summary.num_tests_passed = 0;
 
+% stash user's formatting
+if (is_octave)
+  if (compare_versions(OCTAVE_VERSION(), '4.3.0', '>='))
```

Comment:
Recommend changing this to `4.4.0`, 4.3.0 was a development version, shouldn't exist anywhere now

Reply:
Wouldn't it still be more logical to test for 4.3.0 since the "old" way of doing it will fail there (if I understand https://github.com/cbm755/octsympy/issues/713#issuecomment-269240561 right)?

Reply:
IIRC, there are probably some range of commits where neither works...

People running old dev versions get to keep both pieces :-)

Or we could bump our min version to 4.4 and simplify this...?

Reply:
I added a try-catch version with no version testing: personally I'm not a fan is this kind of laisez-faire Python-esque code, but it does satisfy both your comments :)